lib/provisioner/worker/plugins/automators/docker_automator/docker_automator.rb

```diff
@@ -153,6 +170,9 @@ def parse_inputmap(inputmap)
     @fields = inputmap['fields']
     @image_name = @fields && @fields.key?('image_name') ? @fields['image_name'].gsub(/\s+/, '') : nil
     @command = @fields && @fields.key?('command') ? @fields['command'] : nil
+    @envs = @fields && @fields.key?('environment_variables') ? @fields['environment_variables'] : nil
```

Comment:
there's still NPE potential [here](https://github.com/caskdata/coopr-provisioner/blob/13a32be7895c276498b37215dc7a8a21d9fab8cf/lib/provisioner/worker/plugins/automators/docker_automator/docker_automator.rb#L121) if ``@envs`` gets set to nil.

I suggest initializing these to arrays, empty or not:
```
@envs = @fields && @fields.key?('environment_variables') ? @fields['environment_variables'].split(',') : []
```
then the ``envmap`` method can simply be:
```
@envs.map {|x| "-e #{x}" }.join(' ')
```
(returns ``""`` if ``@envs`` is ``[]``)

same for ``@links`` and ``@vols``